components/omega/doc/userGuide/MetaData.md

```diff
@@ -0,0 +1,245 @@
+(omega-user-metadata)=
+
```

Comment:
The user guide should be targeted at those who are running the model and configuring the model for a simulation. Much of the detail currently in this doc should is more appropriate for the developer's guide which is where people who are modifying Omega code will get the necessary details for defining fields.

Reply:
I moved most of the content in user guide to dev guide. In user guide, there are shorten version of it with code examples.